modules/pubsub/src/main/scala/zio/gcp/pubsub/subscriber/Subscriber.scala

```diff
@@ -0,0 +1,33 @@
+package zio.gcp.pubsub.subscriber
+
+import zio._
+import com.google.pubsub.v1.ProjectSubscriptionName
+import com.google.cloud.pubsub.v1.MessageReceiver
+import com.google.api.core.ApiService
+import com.google.api.gax.batching.FlowControlSettings
+import com.google.cloud.pubsub.v1.{ Subscriber => Subscriber_ }
```

Comment:
I'd rather prefix it, e.g. `GSubscriber`.

Reply:
amended as per suggestion